components/twitter/twitter.app.js

```diff
@@ -810,6 +870,13 @@ module.exports = {
     },
     async _makeRequest(config) {
       if (!config.headers) config.headers = {}
+      if (config.params) {
+        const query = querystring.stringify(config.params)
+        delete config.params
+        const sep = config.url.indexOf('?') === -1 ? '?' : '&'
+        config.url += `${sep}${query}`
+        config.url = config.url.replace('?&','?')
+      }
       const authorization = await this._getAuthorizationHeader(config)
```

Comment:
This is the call I think is 502ing. You should try/catch and decide if it fails whether you should just bail on this request or retry it. Bailing only works if the other methods are okay with this not returning a proper axios response

Reply:
note: add sleep in the catch